op/nodedns/nodedns.go

```diff
@@ -32,8 +32,20 @@ server:
   log-replies: yes
   log-local-actions: yes
   log-servfail: yes
+  num-threads: 4
+  incoming-num-tcp: 1000
+  outgoing-num-tcp: 1000
+  num-queries-per-thread: 4096
+  outgoing-range: 8192
   rrset-roundrobin: yes
   extended-statistics: yes
+  rrset-cache-size: 256m
+  rrset-cache-slabs: 2
```

Comment:
```suggestion
  rrset-cache-slabs: 4
```

I suspect `*-slabs` should be 4 when `num-threads` is 4?
A power of two means a number of the form `2^N`, not `N`.